packages/admin-ui/src/filters/filter-status.tsx

```diff
@@ -32,11 +33,11 @@ export function FilterStatus(props: FilterStatusProps) {
       {status === 'loading' && (
         <Stack csx={style.statusLayout} space="$space-6">
           <Stack space="$space-4" csx={{ width: '3/4' }}>
-            <Skeleton csx={{ height: 20 }} />
-            <Skeleton csx={{ width: '9/12', height: 20 }} />
-            <Skeleton csx={{ width: '10/12', height: 20 }} />
-            <Skeleton csx={{ width: '11/12', height: 20 }} />
-            <Skeleton csx={{ width: '8/12', height: 20 }} />
+            <Skeleton className={csx({ height: 20 })} />
+            <Skeleton className={csx({ width: '9/12', height: 20 })} />
+            <Skeleton className={csx({ width: '10/12', height: 20 })} />
+            <Skeleton className={csx({ width: '11/12', height: 20 })} />
+            <Skeleton className={csx({ width: '8/12', height: 20 })} />
```

Comment:
Suggestion: use `'1.25rem'` or equivalent token.

```suggestion
            <Skeleton className={csx({ width: '9/12', height: '1.25rem' })} />
            <Skeleton className={csx({ width: '10/12', height: '1.25rem'  })} />
            <Skeleton className={csx({ width: '11/12', height: '1.25rem'  })} />
            <Skeleton className={csx({ width: '8/12', height: '1.25rem'  })} />
```

Reply:
Since those px behavior are in other components, I would let to change it on the respective components, what do you think?

Reply:
Sure! Do it 🚀